symfony/plugins/orangehrmAttendancePlugin/Controller/EditAttendanceController.php

```diff
@@ -34,6 +34,7 @@ class EditAttendanceController extends AbstractVueController
      */
     public function preRender(Request $request): void
     {
+        // TODO: Check if user can edit attendance record
```

Comment:
@SandeepaOHRM @vageeshorangehrm can you help on this

Reply:
yes

Reply:
I will help him

Reply:
fixed in [7353113](https://github.com/orangehrm/orangehrm/pull/1135/commits/7353113283824d3793e07e3b1e1ab3eb61e8e55f)